Packs/CommonScripts/Scripts/CreateIndicatorsFromSTIX/CreateIndicatorsFromSTIX.yml

```diff
@@ -2,6 +2,15 @@ args:
 - description: The entry ID of the STIX file.
   name: entry_id
   required: true
+- auto: PREDEFINED
+  defaultValue: "false"
+  description: Adds Indicators to context.
+  name: add_context
+  predefined:
+  - "true"
+  - "false"
+- description: Adds tags to Indicators. Comma separated.
+  name: tags
```

Comment:
please add the `StixIndicators.type` and `StixIndicators.value` to the outputs

Reply:
done as per commit.